openghg/data/process_gcwerks_parameters.json

```diff
@@ -1,47 +1,47 @@
 {
     "GCWERKS": {
         "instruments": {
-            "GCMD": "",
-            "GCMS": "",
+            "gcmd": "",
+            "gcms": "",
```

Comment:
Quick question: will the ObsSurface.read_file() function still accept instruments as upper case? Or will they need to lower case now?